samza-core/src/main/scala/org/apache/samza/storage/TaskStorageCommitManager.java

```diff
@@ -65,9 +61,9 @@ public void start() {
   /**
    * Commits the local state on the remote backup implementation
    * TODO BLOCKER dchen add comments / docs for what all these Map keys and value are.
-   * @return Committed StoreName to StateCheckpointMarker mappings of the committed SSPs
+   * @return Committed Map of FactoryName to (Map of StoreName to StateCheckpointMarker) mappings of the committed SSPs
```

Comment:
Clean up redundant description. (Committed map of .. mappings of the committed SSPs)